$OpenBSD: patch-wscript,v 1.1.1.1 2012/07/17 09:55:59 jasper Exp $

Do not build against gtk or gdk. Check glib package.
Commit 0df8a8f7e7ee2f88acaf09848c31615da35406cc

--- wscript.orig	Mon Jul 16 21:37:57 2012
+++ wscript	Mon Jul 16 21:38:37 2012
@@ -13,9 +13,7 @@ def configure(conf):
   conf.check_tool("compiler_cxx")
   conf.check_tool("node_addon")
   conf.check_cfg(package='gobject-introspection-1.0', uselib_store='GIREPOSITORY', args='--cflags --libs')
-  conf.check_cfg(package='glib', uselib_store='GLIB', args='--cflags --libs')
-  conf.check_cfg(package='gtk+-2.0', uselib_store='GTK', args='--cflags --libs')
-  conf.check_cfg(package='gdk-2.0', uselib_store='GDK', args='--cflags --libs')
+  conf.check_cfg(package='glib-2.0', uselib_store='GLIB', args='--cflags --libs')
 
 def build(bld):
   obj = bld.new_task_gen("cxx", "shlib", "node_addon")
@@ -30,7 +28,7 @@ def build(bld):
     "src/types/function.cc"
   ]
   obj.cxxflags = ["-D_FILE_OFFSET_BITS=64", "-D_LARGEFILE_SOURCE"]
-  obj.uselib = "GIREPOSITORY GLIB GTK GDK"
+  obj.uselib = "GIREPOSITORY GLIB"
 
 def shutdown():
   if Options.commands['clean']:
